backend/src/routes/user/user.router.ts

```diff
@@ -0,0 +1,34 @@
+import { Router } from 'express';
+
+const PATH = '/users';
+
+const userRouter = Router();
+
```

Comment:
Leave a comment that we need to remove this mock when we create this endpoint.

Reply:
Good point, thanks.